docs/configuration/ai-analysis.rst

```diff
@@ -299,36 +299,318 @@ Advanced - Customizing HolmesGPT
+
+This toolset provides diagnostics for Kubernetes clusters, helping developers identify and resolve issues.
+
+
+.. code-block:: yaml
+
+    toolsets:
+      kubernetes/diagnostics:
+        description: "Advanced diagnostics and troubleshooting tools for Kubernetes clusters"
+        docs_url: "https://kubernetes.io/docs/home/"
+        icon_url: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRPKA-U9m5BxYQDF1O7atMfj9EMMXEoGu4t0Q&s"
+        tags:
+          - core
+        prerequisites:
+          - command: "kubectl version --client"
+        tools:
+          - name: "kubectl_node_health"
+            description: "Check the health status of all nodes in the cluster."
+            command: "kubectl get nodes -o wide"
+          - name: "kubectl_troubleshoot_pod"
+            description: "Fetch logs and describe output for a problematic pod."
+            command: |
+              echo "Logs:" &&
+              kubectl logs {{ pod_name }} -n {{ namespace }} &&
+              echo "\nDescription:" &&
+              kubectl describe pod {{ pod_name }} -n {{ namespace }}
+          - name: "kubectl_check_resource_quota"
+            description: "Fetch the resource quota for a specific namespace."
+            command: "kubectl get resourcequota -n {{ namespace }} -o yaml"
+          - name: "kubectl_pod_disk_usage"
+            description: "Check the disk usage of a specific pod."
+            command: |
+              kubectl exec -n {{ namespace }} {{ pod_name }} -- du -sh /
+          - name: "kubectl_find_evicted_pods"
+            description: "List all evicted pods in a specific namespace."
+            command: "kubectl get pods -n {{ namespace }} --field-selector=status.phase=Failed | grep Evicted"
+          - name: "kubectl_drain_node"
+            description: "Drain a node safely by evicting all pods."
+            command: "kubectl drain {{ node_name }} --ignore-daemonsets --force --delete-emptydir-data"
+
+
+Adding Custom Tools to Holmes
+-----------------------------
+As an example, let's add custom toolset named ``http_tools`` that  makes requests to ``example.com``
+
+.. code-block:: yaml
+
+    enableHolmesGPT: true
+    holmes:
+      additionalEnvVars:
+        - name: ROBUSTA_AI
+          value: "true"
+      toolsets:
+        http_tools:
+          description: "A simple toolset for HTTP requests to example.com"
+          docs_url: "https://example.com"
+          icon_url: "https://example.com/favicon.ico"
+          tags:
+            - cluster
+          prerequisites:
+            - command: "curl -o /dev/null -s -w '%{http_code}' https://example.com "
+              expected_output: "200"
+          tools:
+            - name: "curl_example"
+              description: "Perform a GET request to example.com"
+              command: "curl -X GET https://example.com"
+            - name: "curl_with_params"
+              description: "Perform a GET request to example.com with query parameters"
```

Comment:
I would give a description that helps the LLM understand when to use this (use this to xyz...)

Reply:
Updated descriptions